src/sentry/tagstore/snuba/backend.py

```diff
@@ -753,3 +754,9 @@ def update_group_for_events(self, project_id, event_ids, destination_id):
     def update_group_tag_key_values_seen(self, project_id, group_ids):
         # Called by ``unmerge``. The return value is not used.
         pass
+
+    def delay_index_event_tags(self, organization_id, project_id, group_id,
+                               environment_id, event_id, tags, date_added):
+        # Called by ``EventManager.save``. The return value is not
+        # used.
+        raise NotImplementedError
```

Comment:
Won't this get bubbled up into event manager? Or is NotImplementedError handled somewhere int he service delegator stuff automatically?

Reply:
Yes, I should really have proofread this before submitting

Reply:
Thanks, this is fixed with a73d15f